contrib/platform/src/com/sun/jna/platform/win32/Guid.java

```diff
@@ -466,16 +468,14 @@ public CLSID(GUID guid) {
 
     /**
      * The Class REFIID.
-     *
-     * @author Tobias Wolf, wolf.tobias@gmx.net
      */
-    public class REFIID extends IID {
+    public class REFIID extends ByReference {
```

Comment:
This only makes a difference when the `Structure` is used as a field within a `Structure`.  It's better to leave the main definition untagged and use an explicitly tagged inner class which derives from the main one.